# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1372455958 -7200
#      Fri Jun 28 23:45:58 2013 +0200
# Node ID 06657ee17aa2e127edd74f2597062561bb0a4490
# Parent  c82b3614eacea4775865c8e82c4dd2353c0e4fc4
* nl_NL.ts: updated dutch translation by Philip

diff --git a/libgui/languages/nl_NL.ts b/libgui/languages/nl_NL.ts
--- a/libgui/languages/nl_NL.ts
+++ b/libgui/languages/nl_NL.ts
@@ -9,62 +9,62 @@
         <translation type="unfinished">Alles selecteren</translation>
     </message>
 </context>
 <context>
     <name>QObject</name>
     <message>
         <location filename="../src/workspace-model.cc" line="+75"/>
         <source>automatic</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Automatisch</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>function</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Functie</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>global</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Gobaal</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>hidden</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Verborgen</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>inherited</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Overgeërfd</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>persistent</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Blijvend</translation>
     </message>
     <message>
         <location filename="../qterminal/libqterminal/QTerminal.cc" line="+64"/>
         <source>foreground</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Voorgrond</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>background</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Achtergrond</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>selection</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Selectie</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>cursor</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Aanwijzer</translation>
     </message>
 </context>
 <context>
     <name>QTerminal</name>
     <message>
         <location filename="../qterminal/libqterminal/QTerminal.h" line="+116"/>
         <source>Copy</source>
         <translation type="unfinished">Kopiëren</translation>
@@ -80,572 +80,572 @@
         <translation type="unfinished"></translation>
     </message>
 </context>
 <context>
     <name>QWinTerminalImpl</name>
     <message>
         <location filename="../qterminal/libqterminal/win32/QWinTerminalImpl.cpp" line="+1451"/>
         <source>copied selection to clipboard</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">selectie naar klembord gekopieerd</translation>
     </message>
 </context>
 <context>
     <name>QsciLexerBatch</name>
     <message>
         <location filename="../../../qsci/qscilexerbatch.cpp" line="+179"/>
         <source>Default</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Standaard</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Comment</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Commentaar</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Keyword</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Trefwoord</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Label</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Etiket</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Hide command character</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Verberg opdrachtteken</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>External command</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Externe opdracht</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Variable</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Variabele</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Operator</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Operator</translation>
     </message>
 </context>
 <context>
     <name>QsciLexerCPP</name>
     <message>
         <location filename="../../../qsci/qscilexercpp.cpp" line="+352"/>
         <source>Default</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Verstekwaarde</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive default</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactieve verstekwaarde</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>C comment</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">C commentaar</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive C comment</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactief C commentaar</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>C++ comment</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">C++ commentaar</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive C++ comment</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactief C++ commentaar</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>JavaDoc style C comment</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">C commentaar in JavaDoc stijl</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive JavaDoc style C comment</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactief C commentaar in JavaDoc stijl</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Number</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Getal</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive number</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactief getal</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Keyword</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Trefwoord</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive keyword</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactief trefwoord</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Double-quoted string</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Tekst met dubbele aanhalingstekens</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive double-quoted string</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactieve tekst met dubbele aanhalingstekens</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Single-quoted string</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Tekst met aanhalingstekens</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive single-quoted string</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactieve tekst met aanhalingstekens</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>IDL UUID</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive IDL UUID</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Pre-processor block</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Preprocessor blok</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive pre-processor block</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactief preprocessor blok</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Operator</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Operator</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive operator</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactieve operator</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Identifier</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive identifier</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactieve identifier</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Unclosed string</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Niet afgesloten tekst</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive unclosed string</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactieve niet afgesloten tekst</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>C# verbatim string</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">C# letterlijke tekst</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive C# verbatim string</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactieve C# letterlijke tekst</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>JavaScript regular expression</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Javascript reguliere uitdrukking</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive JavaScript regular expression</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inatieve Javascript reguliere uitdrukking</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>JavaDoc style C++ comment</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">C++ commentaar in JavaDoc stijl</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive JavaDoc style C++ comment</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactief C++ commentaar in JavaDoc stijl</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Secondary keywords and identifiers</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Secundaire trefwoorden en identifiers</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive secondary keywords and identifiers</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactieve secundaire trefwoorden en identifiers</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>JavaDoc keyword</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">JavaDoc sleutelwoord</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive JavaDoc keyword</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactief JavaDoc sleutelwoord</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>JavaDoc keyword error</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">fout JavaDoc trefwoord</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive JavaDoc keyword error</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactief fout JavaDoc trefwoord</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Global classes and typedefs</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Globale klassen en typedefs</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive global classes and typedefs</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactieve globale klassen en typedefs</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>C++ raw string</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">C++ ruwe tekst</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Inactive C++ raw string</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Inactieve C++ ruwe tekst</translation>
     </message>
 </context>
 <context>
     <name>QsciLexerDiff</name>
     <message>
         <location filename="../../../qsci/qscilexerdiff.cpp" line="+107"/>
         <source>Default</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Verstekwaarde</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Comment</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Commentaar</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Command</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Opdracht</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Header</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Koptekst</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Position</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Positie</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Removed line</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Verwijderde regel</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Added line</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Toegevoegde regel</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Changed line</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Gewijzigde regel</translation>
     </message>
 </context>
 <context>
     <name>QsciLexerMatlab</name>
     <message>
         <location filename="../../../qsci/qscilexermatlab.cpp" line="+138"/>
         <source>Default</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Verstekwaarde</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Comment</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Commentaar</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Command</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Opdracht</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Number</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Getal</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Keyword</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Trefwoord</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Single-quoted string</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Tekst tussen aanhalingstekens</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Operator</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Operator</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Identifier</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Identifier</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Double-quoted string</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Tekst tussen dubbele aanhalingstekens</translation>
     </message>
 </context>
 <context>
     <name>QsciLexerPerl</name>
     <message>
         <location filename="../../../qsci/qscilexerperl.cpp" line="+333"/>
         <source>Default</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Verstekwaarde</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Error</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Fout</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Comment</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Commentaar</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>POD</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">POD</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Number</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Getal</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Keyword</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Trefwoord</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Double-quoted string</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Tekst tussen dubbele aanhalingstekens</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Single-quoted string</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Tekst tussen aanhalingstekens</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Operator</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Operator</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Identifier</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Identifier</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Scalar</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Enkel getal</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Array</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Array</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Hash</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Hash</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Symbol table</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Symbooltabel</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Regular expression</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Reguliere uitdrukking</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Substitution</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Vervanging</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Backticks</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Data section</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Data sectie</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Here document delimiter</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Here document  scheidingsteken</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Single-quoted here document</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Here document tussen aanhalingstekens</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Double-quoted here document</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Here dicument tussen dubbele aanhalingstekens</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Backtick here document</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Backtick here document</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Quoted string (q)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">q tekst tussen aanhalingstekens</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Quoted string (qq)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">qq tekst tussen aanhalingstekens</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Quoted string (qx)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">qx tekst tussen aanhalingstekens</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Quoted string (qr)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">qr tekst tussen aanhalingstekens</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Quoted string (qw)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">qw tekst tussen aanhalingstekens</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>POD verbatim</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">POD letterlijk</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Subroutine prototype</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Format identifier</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Sjabloon identifier</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Format body</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Sjabloon inhoud</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Double-quoted string (interpolated variable)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Tekst tussen aanhalingstekens (geïnterpoleerde variabele)</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Translation</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Vertaling</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Regular expression (interpolated variable)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Reguliere uitdrukking (Geïnterpoleerde variabele)</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Substitution (interpolated variable)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Vervanging</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Backticks (interpolated variable)</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Double-quoted here document (interpolated variable)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Here document tussen dubbele aanhalingstekens (geïnterpoleerde variabele</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Backtick here document (interpolated variable)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Backtick here document (geïnterpoleerde variabele)</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Quoted string (qq, interpolated variable)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Tekst tussen aanhalingstekens (qq, geïnterpoleerde variabele)</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Quoted string (qx, interpolated variable)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Tekst tussen aanhalingstekens (qx, geïnterpoleerde variabele)</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Quoted string (qr, interpolated variable)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Tekst tussen aanhalingstekens (qr, geïnterpoleerde variabele)</translation>
     </message>
 </context>
 <context>
     <name>QsciScintilla</name>
     <message>
         <location filename="../../../qsci/qsciscintilla.cpp" line="+4201"/>
         <source>&amp;Undo</source>
         <translation type="unfinished">Ongedaan maken</translation>
@@ -663,27 +663,27 @@
     <message>
         <location line="+5"/>
         <source>&amp;Copy</source>
         <translation type="unfinished">Kopiëren</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>&amp;Paste</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Plakken</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Delete</source>
         <translation type="unfinished">Verwijderen</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Select All</source>
-        <translation type="unfinished">Alles selecteren</translation>
+        <translation type="unfinished">Alles kiezen</translation>
     </message>
 </context>
 <context>
     <name>documentation_dock_widget</name>
     <message>
         <location filename="../src/documentation-dock-widget.cc" line="+34"/>
         <source>Documentation</source>
         <translation>Documentatie</translation>
@@ -717,17 +717,17 @@
     </message>
     <message>
         <location line="+49"/>
         <source>File not saved! A file with the selected name
 %1
 is already open in the editor</source>
         <translation>Bestand niet opgeslagen! Een bestand met de naam
 %1
-is al geopend in de editor</translation>
+is reeds geopend in de editor</translation>
     </message>
     <message>
         <location line="+28"/>
         <source>The associated file editor tab has disappeared.  It was likely closed by some means.</source>
         <translation>Het bijbehorende tabblad in de editor is verdwenen. Het is waarschijnlijk op een of andere manier gesloten.</translation>
     </message>
     <message>
         <location line="+205"/>
@@ -802,32 +802,32 @@ is al geopend in de editor</translation>
     <message>
         <location line="+3"/>
         <source>&amp;Remove All Bookmarks</source>
         <translation>Alle bladwijzers ve&amp;rwijderen</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>&amp;Next breakpoint</source>
-        <translation>Volge&amp;nd breekpunt</translation>
+        <translation>Volge&amp;nd onderbreekpunt</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Pre&amp;vious breakpoint</source>
-        <translation>&amp;Vorig breekpunt</translation>
+        <translation>&amp;Vorig onderbreekpunt</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Toggle &amp;breakpoint</source>
-        <translation>&amp;Breekpunt invoegen</translation>
+        <translation>&amp;Onderbreekpunt invoegen</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Remove All breakpoints</source>
-        <translation>Alle breekpunten ve&amp;rwijderen</translation>
+        <translation>Alle onderbreekpunten ve&amp;rwijderen</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Comment</source>
         <translation type="unfinished">Zet om naar commentaar</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -890,37 +890,37 @@ is al geopend in de editor</translation>
         <translation>Uitvoe&amp;ren</translation>
     </message>
 </context>
 <context>
     <name>file_editor_tab</name>
     <message>
         <location filename="../src/m-editor/file-editor-tab.cc" line="+726"/>
         <source>Goto line</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Ga naar regel ...</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Line number</source>
         <translation type="unfinished">Regelnummer</translation>
     </message>
     <message>
         <location line="+70"/>
         <source>&lt;unnamed&gt;</source>
         <translation type="unfinished">naamloos</translation>
     </message>
     <message>
         <location line="+40"/>
         <source>Do you want to save or discard the changes?</source>
-        <translation type="unfinished">Wil je wijzigingen mee opslaan of niet?</translation>
+        <translation type="unfinished">Wil je de wijzigingen opslaan of niet?</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Do you want to cancel closing, save or discard the changes?</source>
-        <translation type="unfinished">Wil je niet meer afsluiten, het gewijzigde of het originele bestand opslaan?</translation>
+        <translation type="unfinished">Wil je niet meer afsluiten, het gewijzigde, of het originele bestand opslaan?</translation>
     </message>
     <message>
         <location line="+4"/>
         <location line="+114"/>
         <location line="+104"/>
         <location line="+66"/>
         <location line="+22"/>
         <source>Octave Editor</source>
@@ -1015,17 +1015,17 @@ is gewist of hernoemd. Wil je het nu ops
     <message>
         <location line="+4"/>
         <source>Set octave directory</source>
         <translation type="unfinished">Stel octave map in</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Set octave directroy to current browser directory</source>
-        <translation type="unfinished">Stel octave map in op huidige browser map</translation>
+        <translation type="unfinished">Maak huidige browser map de Octave map</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Actions on current directory</source>
         <translation type="unfinished">Bewerkingen op huidige map</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -1123,50 +1123,51 @@ is gewist of hernoemd. Wil je het nu ops
         <location line="+11"/>
         <source>Delete file/directory</source>
         <translation type="unfinished">Wis bestand/map</translation>
     </message>
     <message>
         <location line="-10"/>
         <source>Are you sre you want to delete
 </source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Weet u zeker dat u dit wilt verwijderen
+        </translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Can not delete a directory that is not empty</source>
         <translation type="unfinished">Kan geen niet-lege map wissen</translation>
     </message>
     <message>
         <location line="+128"/>
         <source>Set directory of file browser</source>
         <translation type="unfinished">Stel file browser map in</translation>
     </message>
     <message>
         <location line="+27"/>
         <source>Create File</source>
-        <translation type="unfinished">Nieuw bestand</translation>
+        <translation type="unfinished">Maak nieuw bestand</translation>
     </message>
     <message>
         <location line="+0"/>
         <source>Create file in
 </source>
-        <translation type="unfinished">Nieuw bestand in
+        <translation type="unfinished">Maak nieuw bestand in
 </translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Create Directory</source>
-        <translation type="unfinished">Nieuwe map</translation>
+        <translation type="unfinished">Maak nieuwe map</translation>
     </message>
     <message>
         <location line="+0"/>
         <source>Create folder in
 </source>
-        <translation type="unfinished">Nieuwe map in
+        <translation type="unfinished">Maak nieuwe map in
 </translation>
     </message>
 </context>
 <context>
     <name>find_dialog</name>
     <message>
         <location filename="../src/m-editor/find-dialog.cc" line="+77"/>
         <source>Find &amp;what:</source>
@@ -1185,17 +1186,17 @@ is gewist of hernoemd. Wil je het nu ops
     <message>
         <location line="+1"/>
         <source>Search from &amp;start</source>
         <translation>Zoek vanaf het begin</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>&amp;Wrap while searching</source>
-        <translation>Bestandseinde negeren</translation>
+        <translation>Na bestandseinde doorgaan aan begin</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Find Next</source>
         <translation>Volgende zoeken</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1323,22 +1324,22 @@ is gewist of hernoemd. Wil je het nu ops
     <message>
         <location line="+2"/>
         <source>Include matching directories in search results</source>
         <translation type="unfinished">Neem overeenkomende mappen mee in zoekresultaten</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Name case insensitive</source>
-        <translation type="unfinished">Niet hoofdlettergevoelige naam</translation>
+        <translation type="unfinished">Niet-hoofdlettergevoelige naam</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Set matching name is case insensitive</source>
-        <translation type="unfinished">Naam instellen op niet hoofdlettergevoelig</translation>
+        <translation type="unfinished">Naam instellen op niet-hoofdlettergevoelig</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Contains text:</source>
         <translation type="unfinished">Met tekst:</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1348,22 +1349,22 @@ is gewist of hernoemd. Wil je het nu ops
     <message>
         <location line="+4"/>
         <source>Text to match</source>
         <translation type="unfinished">Overeen te komen tekst</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Text case insensitive</source>
-        <translation type="unfinished">Niet hoofdlettergevoelige tekst</translation>
+        <translation type="unfinished">Niet-hoofdlettergevoelige tekst</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Set text content is case insensitive</source>
-        <translation type="unfinished">Tekst instellen op niet hoofdlettergevoelig</translation>
+        <translation type="unfinished">Tekst instellen op niet-hoofdlettergevoelig</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Search results</source>
         <translation type="unfinished">Zoekresultaten</translation>
     </message>
     <message>
         <location line="+11"/>
@@ -1449,17 +1450,17 @@ is gewist of hernoemd. Wil je het nu ops
     <message>
         <location line="+20"/>
         <source>Copy</source>
         <translation>Kopiëren</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Evaluate</source>
-        <translation>Voer uit</translation>
+        <translation>Werk uit</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Create script</source>
         <translation type="unfinished">Nieuw script</translation>
     </message>
 </context>
 <context>
@@ -1619,17 +1620,17 @@ is gewist of hernoemd. Wil je het nu ops
     <message>
         <location line="+3"/>
         <source>Online</source>
         <translation type="unfinished">Online</translation>
     </message>
     <message>
         <location line="+30"/>
         <source>Enter directory name</source>
-        <translation type="unfinished">Voer mapnaam in</translation>
+        <translation type="unfinished">Geef mapnaam op</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Current Directory: </source>
         <translation type="unfinished">Huidige map:</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -1738,17 +1739,17 @@ is gewist of hernoemd. Wil je het nu ops
     <message>
         <location line="+82"/>
         <source>Dock widget</source>
         <translation type="unfinished">Widget in venster opnemen</translation>
     </message>
     <message>
         <location line="+29"/>
         <source>Unock widget</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Laat widget zweven</translation>
     </message>
 </context>
 <context>
     <name>octave_qscintilla</name>
     <message>
         <location filename="../src/m-editor/octave-qscintilla.cc" line="+85"/>
         <source>help</source>
         <translation type="unfinished">hulp</translation>
@@ -1857,17 +1858,17 @@ is gewist of hernoemd. Wil je het nu ops
     <message>
         <location line="+7"/>
         <source>Show indentation guides</source>
         <translation type="unfinished">Laat inspringraster zien</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Backspace unindents line</source>
-        <translation type="unfinished">Backspace laat terug inspringen</translation>
+        <translation type="unfinished">Backspace doet terug inspringen</translation>
     </message>
     <message>
         <location line="+84"/>
         <source>Characters before list with suggestions is displayed</source>
         <translation type="unfinished">Aantal tekens voordat lijst met suggesties wordt weergegeven</translation>
     </message>
     <message>
         <location line="+71"/>
@@ -2047,17 +2048,17 @@ is gewist of hernoemd. Wil je het nu ops
     <message>
         <location line="+7"/>
         <source>Language (requires restart)</source>
         <translation type="unfinished">Taal instellen (vereist herstart)</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Icon size</source>
-        <translation type="unfinished">Iccongrootte</translation>
+        <translation type="unfinished">Icoongrootte</translation>
     </message>
     <message>
         <location line="+1098"/>
         <source>Socks5Proxy</source>
         <translation></translation>
     </message>
     <message>
         <location line="-16"/>
@@ -2084,27 +2085,27 @@ is gewist of hernoemd. Wil je het nu ops
         <location line="+4"/>
         <location line="+334"/>
         <source>System setting</source>
         <translation type="unfinished">Systeeminstelling</translation>
     </message>
     <message>
         <location line="-268"/>
         <source>IBeam Cursor</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">I-balkje</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Block Cursor</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Blokje</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Underline Cursor</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Liggend streepje</translation>
     </message>
     <message>
         <location line="+129"/>
         <source>Difference to the defalt size</source>
         <translation type="unfinished">Verschil met standaardgrootte</translation>
     </message>
     <message>
         <location line="+6"/>
@@ -2222,17 +2223,17 @@ is gewist of hernoemd. Wil je het nu ops
     <message>
         <location line="+1"/>
         <source>Storage Class</source>
         <translation type="unfinished">Klasse</translation>
     </message>
     <message>
         <location line="+107"/>
         <source>Right click to copy, rename, or display</source>
-        <translation type="unfinished">Klik rechts voor kopiëren, hernoemen of weergeven</translation>
+        <translation type="unfinished">Klik rechts om te kopiëren, hernoemen of weergeven</translation>
     </message>
 </context>
 <context>
     <name>workspace_view</name>
     <message>
         <location filename="../src/workspace-view.cc" line="+47"/>
         <source>Workspace</source>
         <translation>Werkruimte</translation>
